glossary.md

```diff
@@ -1,22 +1,149 @@
+* **LE**: Little [Endian][] - A Byte Order where the smallest bit comes first. The opposite of **BE**.
+* **LGTM/SGTM**: Looks/Sounds good to me - Commonly used to approve a code review.
+* **LTS**: [Long Term Support][] - Support provided for a software version for an extended period.
+* **MDN**: [Mozilla Development Network][] - Resource for web developers.
+* **MVC**: [Model-View-Controller][] - A software design pattern commonly used for developing user interfaces. It
+  separates the application into three interconnected components: the model (data), the view (presentation), and the
+  controller (logic).
+* **Native modules/addons**: Modules implemented in C/C++ that expose interfaces callable from JS.
+* **NPM**: [NPM][] - A package manager for JavaScript, widely used for managing dependencies in
+  Node.js projects and for sharing code with others.
+* **OOB**: Out Of Bounds - Used in the context of array access.
+* **OOM**: Out Of Memory - Situation where a computer program exceeds its memory allocation.
+* **OOP**: [Object-Oriented Programming][] - A programming paradigm based on the concept of "objects," which can
+  contain data and code to manipulate that data. OOP languages include features such as encapsulation, inheritance,
+  and polymorphism.
+* **PPC**: [PowerPC][] - A type of microprocessor architecture.
+* **Primordials**: Pristine built-ins in JavaScript that are not affected by prototype pollution.
+* **Prototype Pollution**: Process in which object prototypes are mutated via user action.
+* **RAII**: [Resource Acquisition Is Initialization][] - Programming idiom used to manage resources in C++.
+* **REPL**: [Read Evaluate Print Loop][] - Environment for interactive programming.
+* **RFC**: [Request For Comments][] - A Document used in standardization processes.
+* **RSLGTM/RSSGTM**: Rubber-Stamp Looks/Sounds Good To Me - The reviewer approves without doing a full code review.
+* **RSS**: [Resident Set Size][] - Amount of memory occupied by a process in RAM.
+* **SMP**: [Symmetric Multi-Processor][] - Architecture where multiple processors share the same memory.
+* **Snapshot**: Chunk of bytes containing data serialized from a V8 heap.
+* **TBH**: To Be Honest.
+* **TSC**: Technical Steering Committee - Governing body within a project.
+* **UI**: [User Interface][] - The point of interaction between a user and a computer program. It includes elements
+  such as buttons, menus, and other graphical elements that allow users to interact with the software.
+* **URL**: [Uniform Resource Locator][] - A reference to a web resource that specifies its location on a computer
+  network and the mechanism for retrieving it, typically using the HTTP or HTTPS protocol.
+* **UTF-8**: [Unicode Transformation Format - 8-bit][] - A variable-width character encoding widely used for
+  representing Unicode characters efficiently in byte-oriented systems.
+* **V8**: [The JavaScript engine][] that powers Node.js and Chrome browser.
```

Comment:
It powers more than just Node.js and Chrome, and since I don't think we want to maintain a list, let's only mention Node.js.
```suggestion
* **V8**: [The JavaScript and WebAssembly engine][] that powers Node.js.
```